ipfs-cluster-service/main.go

```diff
@@ -193,6 +193,10 @@ func main() {
 			Value: "disk-freespace",
 			Usage: "allocation strategy to use [disk-freespace,disk-reposize,numpin].",
 		},
+		cli.BoolFlag{
```

Comment:
I think I like this better as a flag specific for the `daemon` subcommand, rather than a global. `ipfs-cluster-service daemon --ugprade`